src/tools/language-server/language-service.ts

```diff
@@ -8,90 +8,105 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {ErrorCodes, Message, StreamMessageReader as VSCodeStreamMessageReader, StreamMessageWriter as VSCodeStreamMessageWriter} from 'vscode-jsonrpc';
-import {isNotificationMessage, isRequestMessage, isResponseMessage} from 'vscode-jsonrpc/lib/messages';
+import {ExecuteCommandParams, InitializeParams, IConnection, Diagnostic, DiagnosticSeverity, DidChangeTextDocumentParams, DidOpenTextDocumentParams, DidSaveTextDocumentParams, TextDocumentSyncKind, InitializeResult} from 'vscode-languageserver';
```

Comment:
Is there a better formatting for this, or is it crazy to use '*'? Similar for util on next line.

Reply:
I prefer not to use star as I then don't know where a name comes from without starting up some more dev tooling.

I can line wrap these (and have stopped using some, these will be removed).

Reply:
You can use `* as Foo` if you wanted. (But I don't mind the long import list)